core/types.go

```diff
@@ -169,9 +170,45 @@
 	return resp, nil
 }
 
-// Signature is a BLS12-381 Signature.
+// NewPartialSignature is a convenience function that returns a new partially signature.
+func NewPartialSignature(sig Signature, shareIdx int) ParSignedData2 {
+	return ParSignedData2{
+		SignedData: sig,
+		ShareIdx:   shareIdx,
+	}
+}
+
+// Signature is a BLS12-381 Signature. It implements SignedData.
 type Signature []byte
```

Comment:
If this is an implementation of SignedData then why it is not present in signeddata.go?